docs/content/doc/developers/hacking-on-gitea.en-us.md

```diff
@@ -309,6 +309,19 @@ Visual Studio Code. Look at
 [`contrib/ide/README.md`](https://github.com/go-gitea/gitea/blob/main/contrib/ide/README.md)
 for more information.
 
+## GoLand
+
+Clicking the `Run Application` arrow on the function `func main()` in `/main.go` 
+can quickly start a debuggable gitea instance.
+
+The `Output Directory` in `Run/Debug Configuration` MUST be set to the 
+gitea project directory (which contains `main.go` and `go.mod`), 
+otherwise, the started instance's working directory is a GoLand's temporary directory 
+and prevents gitea from loading dynamic resources (eg: templates) in a development environment.  
```

Comment:
Ah if that's the only problem then you could consider setting one of the custom_dir or other environment variables. But this sounds easier overall.